pkgs/dartpad_ui/lib/model/model.dart

```diff
@@ -14,6 +14,12 @@ import '../primitives/gists.dart';
 import '../primitives/samples.g.dart';
 import '../primitives/utils.dart';
 
+/// A compile-time flag to control making requests over websockets.
+///
+/// Do not check this in `true`; this will create a long-lived connection to the
+/// backend and we don't yet know how well that will scale.
+const useWebsockets = false;
```

Comment:
nit: consider putting this in a separate library similar to the `genAiEnabled` constant

Reply:
sounds good, will do